gateway/1.4/modules/ROOT/pages/_partials/reference-install-next-steps.adoc

```diff
@@ -1,5 +1,24 @@
-Now that you have installed Flex Gateway, you need to register it in either
+
+//for Linux, download and install
+//tag::flex-download-install[]
+After downloading and installing, register Flex Gateway in either
+xref:flex-conn-reg-run.adoc[Connected] or xref:flex-local-reg-run.adoc[Local Mode]. 
+//end::flex-download-install[]
+
+//for Docker and k8, download only
+//tag::flex-k8-download-install[]
+After downloading the image, register Flex Gateway in either
 xref:flex-conn-reg-run.adoc[Connected] or xref:flex-local-reg-run.adoc[Local Mode]. 
+//end::flex-k8-download-install[]
+
+//for Docker and Linux on connected vs. local
+//tag::flex-connected-vs-local[]
+For more detail, see xref:index.adoc#connected-mode-vs-local-mode[Connected Mode vs. Local Mode].
+//end::flex-connected-vs-local[]
+
+//for k8 on connected vs. local
+//tag::flex-connected-vs-local-k8[]
+For more detail, see xref:flex-gateway-k8-overview.adoc[].
+//end::flex-connected-vs-local-k8[]
+
 
-See xref:flex-gateway-getting-started.adoc[Getting Started with Flex Gateway]
```

Comment:
Was Getting Started intentional here? I don't see any info about the difference between the modes on that page.

Reply:
retaining the change